### PR TITLE
Media library: Add functionality to open the modal of collections and add Collections Home View

### PR DESCRIPTION
**Pull request by ghost**
Get Activity Collections from Strapi via API call
Integrate he received data with slack block kit to render Collections on slack home and render the modal of collections

### CODE REVIEW

File: user-interface/modals/daily-checkin-modal.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ async function getDailyCheckinModal(username) {
       },
       label: {
         type: 'plain_text',
-        text: dailyCheckinQuestion.question,
+        text: 'I feel..',
```
Comment (shivamsherje) [inline user-interface/modals/daily-checkin-modal.js:29]: Revert this change.

File: user-interface/modals/content-library-modal.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            type: 'plain_text',
+            text: `*${activity.title}*`,
+            emoji: true,
+          },
+          title_url: 'https://www.youtube.com/embed/RRxQQxiM7AA?feature=oembed&autoplay=1',
+          // description: {
+          //   type: 'plain_text',
+          //   text: `${activity.description}`,
+          //   emoji: true,
+          // },
```
Comment (shivamsherje) [inline user-interface/modals/content-library-modal.js:39]: Remove commented or unwanted code

File: listeners/actions/index.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 const { sampleActionCallback } = require('./sample-action');
+
```
Comment (disha1202) [inline listeners/actions/index.js:2]: Remove unwanted line.

File: user-interface/activity-collection/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-sequences */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-multiple-empty-lines */
+const { getActivityCollection } = require('../../services/strapi/index');
+
+const baseUrl = process.env.STRAPI_BASE_URL;
+
+
+async function activityCollectionView() {
```
Comment (disha1202) [inline user-interface/activity-collection/index.js:9]:
```suggestion
async function getActivityCollectionView() {
```

File: services/strapi/index.js [hunk 5]
```diff
@@ -54,8 +54,24 @@ async function getSurveyQuestions() {
   }
 }
 
+async function getActivityCollection() {
```
Comment (disha1202) [inline services/strapi/index.js:57]:
```suggestion
async function getActivityCollections() {
```

File: user-interface/modals/content-library-modal.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
+/* eslint-disable padded-blocks */
+const { getActivityCollection } = require('../../services/strapi/index');
+
+const baseUrl = process.env.STRAPI_BASE_URL;
+
+// eslint-disable-next-line consistent-return
+async function contentLibraryModal(body) {
+  const collections = await getActivityCollection();
```
Comment (disha1202) [inline user-interface/modals/content-library-modal.js:8]: Why are we fetching all the collections and finding a specific collection when we have access to the collection id. 
We should fetch only a single collection.

File: user-interface/modals/content-library-modal.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
+/* eslint-disable padded-blocks */
+const { getActivityCollection } = require('../../services/strapi/index');
+
+const baseUrl = process.env.STRAPI_BASE_URL;
+
+// eslint-disable-next-line consistent-return
+async function contentLibraryModal(body) {
```
Comment (disha1202) [inline user-interface/modals/content-library-modal.js:7]:
```suggestion
async function getContentLibraryModal(body) {
```

File: user-interface/app-home/unauthorised-user-view.js [hunk 8]
```diff
@@ -1,9 +1,13 @@
 const { getCurrentDay } = require('../../utilities');
 const { getMasterQuote } = require('../../services/strapi/index');
+const { activityCollectionView } = require('../activity-collection/index');
 
 async function unauthorisedUserView() {
   const day = getCurrentDay();
   const quote = await getMasterQuote();
+  let activityCollection = await activityCollectionView();
```
Comment (disha1202) [inline user-interface/app-home/unauthorised-user-view.js:8]:
```suggestion
  let activityCollections = await activityCollectionView();
```

**Review by disha1202 — CHANGES_REQUESTED** [review]

File: listeners/views/index.js [hunk 9]
```diff
@@ -1,9 +1,13 @@
 const { sampleViewCallback } = require('./sample-view');
+const { openDailyCheckinModalCallback } = require('../actions/open-daily-checkin-action');
+const { openContentLibraryModalCallback } = require('../actions/open-content-library-action');
 const { postDailyCheckinFeedback } = require('./daily-checkin');
 const { postSurveyFeedback } = require('./survey');
 
 module.exports.register = (app) => {
   app.view('sample_view_id', sampleViewCallback);
+  app.view('dailyCheckin', openDailyCheckinModalCallback);
```
Comment (disha1202) [inline listeners/views/index.js:9]: Remove this line of code.

**Review by disha1202 — CHANGES_REQUESTED** [review]

File: user-interface/modals/daily-checkin-modal.js (hunk 1 above)
Comment (ghost) [reply user-interface/modals/daily-checkin-modal.js:29]: This is done!

Comment (ghost) [reply user-interface/modals/daily-checkin-modal.js:29]: Check new pr please!
